semcore/notice/src/index.d.ts

```diff
@@ -66,6 +72,8 @@ declare const Notice: Intergalactic.Component<'div', NoticeProps, NoticeContext>
   Label: Intergalactic.Component<'div', NoticeLabelProps>;
   Actions: typeof Box;
   Content: typeof Box;
+  Title: typeof Box;
+  Text: typeof Box;
```

Comment:
You changed the render components, so, change please types

Reply:
Updated